admin/acceptance.py

```diff
@@ -404,14 +409,22 @@ def main(reactor, args, base_path, top_level):
 
     try:
         nodes = yield runner.start_nodes(reactor)
+
+        ca_directory = FilePath(mkdtemp())
+        print("Generating certificates in: {}".format(ca_directory.path))
+        certificates = Certificates.generate(ca_directory, nodes[0].address,
+                                             len(nodes))
+
```

Comment:
Eh, you should care.  "X is slow by Y is slower" is how you get to where we are today.